### ISSUE TITLE
Create ROCK for torchserve-kfs

### ISSUE THREAD

**Issue by kimwnasptd** [canonical/kserve-rocks#2]
Kubeflow is using `pytorch/torchserve-kfs` image, as the KServe runtime image for serving PyTorch models 
https://github.com/kserve/kserve/blob/f7de5e696e8d0e64e3ed2b2493ec64244291a5c9/install/v0.11.0/kserve-runtimes.yaml#L218

The source of this image lives in https://github.com/pytorch/serve/tree/master/kubernetes/kserve and it uses the `pytorch/torchserve` image as base. So in order to construct the ROCK we'll need to build the base torchserve image as well.

IIUC the base `pytorch/torchserve` image is defined and built in https://github.com/pytorch/serve/blob/v0.8.2/docker/Dockerfile

To summarize, our ROCK will need to
* live in a new repo, that aims to provide ROCKs for `pytorch/serve` repo
* Have a first layer that builds the `pytorch/torchserve` image
* Convert the dockerfile of https://github.com/pytorch/serve/tree/master/kubernetes/kserve on top

cc @i-chvets @SamirPS

**Comment by kimwnasptd** [comment]
Also a note regarding versioning and tagging of the upstream `pytorch/torchserve-kfs` image:
* The tag is deduced from the version of pytorch serving runtime, which we can see in tags of the repo
* The KServe python library version can be found in https://github.com/pytorch/serve/blob/master/kubernetes/kserve/requirements.txt#L1

**Comment by mvlassis** [comment]
This issue was completed by https://github.com/canonical/kserve-rocks/pull/106
